runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -92,6 +95,20 @@
      */
     Map<Long, Object> upcallResults;
 
+    /**
+     * Metrics: meter (counter), histogram
+     */
+    private MetricRegistry metrics = CorfuRuntime.getMetrics();
+    private String mpObj = CorfuRuntime.getMpObj();
+    private Timer timerAccess = metrics.timer(mpObj + "access");
+    private Timer timerLogWrite = metrics.timer(mpObj + "log-write");
+    private Timer timerTxn = metrics.timer(mpObj + "txn");
+    private Timer timerUpcall = metrics.timer(mpObj + "upcall");
+    private Counter counterAccessOptimistic = metrics.counter(mpObj + "access-optimistic");
+    private Counter counterAccessLocked = metrics.counter(mpObj + "access-locked");
+    private Counter counterTxnRetry1 = metrics.counter(mpObj + "txn-retry1");
+    private Counter counterTxnRetryN = metrics.counter(mpObj + "txn-retryN");
```

Comment:
what is 1 and what is N? Please use English names. Comments would not hurt either